src/js/videojs.wavesurfer.js

```diff
@@ -253,13 +253,31 @@ class Wavesurfer extends Plugin {
      * @param {string|blob|file} url - Either the URL of the audio file,
      *     a Blob or a File object.
      */
-    load(url) {
-        if (url instanceof Blob || url instanceof File) {
-            this.log('Loading object: ' + JSON.stringify(url));
-            this.surfer.loadBlob(url);
+    load(audioUrl, peakUrl) {
+        let player = this;
```

Comment:
@rbwest can you simply refer to `this` instead of defining and using the `player` var here?

Reply:
and I also prefer `url` over `audioUrl`. Can you also document the `peakUrl` argument?